_i18n/fr/resources/user-guides/ledger-wallet-cli.md

```diff
@@ -0,0 +1,165 @@
+
+6. Vous pourriez avoir besoin de confirmer deux fois avant qu'il ne procède.
+
+7. Votre portefeuille Monero Ledger va maintenant être généré. Notez que cela pourrait prendre de 5 à 10 minutes. De plus, il n'y aura pas de retour immédiat, ni sur la CLI ni sur Ledger.
+
+8. `monero-wallet-cli` va se réactualiser. Patientez jusqu'à la fin de cette opération.
+
+Félicitations, vous pouvez maintenant utiliser votre portefeuille Monero Ledger conjointement avec la CLI.
+
+### 4. Quelques remarques finales
+
+1. Nous vous recommandons fortement de d'abord tester la totalité du processus. C'est à dire, envoyer un petit montant au portefeuille et le restaurer consécutivement (en utilisant le guide susmentionné) pour vérifier que vous pouvez récupérer le portefeuille. Remarquez que, lors de la recréation / restauration d'un portefeuille, vous devez ajouter l'option `--restore-height` (avec la hauteur de bloc de la première transaction de ce portefeuille) à la commande de l'étape 3 (Windows), l'étape 5 (Mac OS X) ou l'étape 3 (Linux). Vous trouverez plus d'information concernant la hauteur de restauration et la manière de l'estimer [ici](https://monero.stackexchange.com/questions/7581/what-is-the-relevance-of-the-restore-height).
+
+2. Si vous utilisez un nœud distant, ajoutez l'option `--daemon-address host:port` à la commande de l'étape 3 (Windows), l'étape 5 (Mac OS X) ou l'étape 3 (Linux).
+
+3. Si vous les souhaitez, vous pouvez personnaliser la valeur de l'option `--subaddress-lookahead`. La première valeur correspond au nombre de comptes et la seconde au nombre de sous-adresses par compte. ainsi, si par exemple vous voulez pré-générer 5 comptes avec 100 sous-adresses chacun, utilisez `--subaddress-lookahead 5:100`. Gardez à l'esprit que plus vous pré-générez de sous-adresses, plus le temps de création de votre portefeuille sur le Ledger sera long.
+
+4. Vous n'avez à utiliser l'option `--generate-from-device` qu'une seule fois (c-à-d lors de la création du portefeuille). Ultérieurement vous l'utiliserez d'une façon similaire à la CLI habituelle, c'est à dire :
+   1. Vous assurer que votre Ledge est connecté et que l'application Monro est démarrée ;
+   2. Ouvrir `monero-wallet-cli`.
+   3. Entrer le nom du portefeuille de votre portefeuille Monero Ledger ;
```

Comment:
This sounds repetitive. How about just `Entrer le nom de votre portefeuille Monero Ledger`?

Reply:
ok